themes/openy_themes/openy_rose/templates/paragraph/paragraph--gallery--default.html.twig

```diff
@@ -80,7 +80,9 @@ set classes = [
         <div class="cta-group">
           <h1>{{ content.field_prgf_headline }}</h1>
           <div class="text">{{ content.field_prgf_description }}</div>
-          <div class="btn btn-default btn-link blue">{{ content.field_prgf_link }}</div>
+          {% if paragraph.field_prgf_link.0.uri %}
```

Comment:
It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level